usr/src/cmd/bhyve/pci_virtio_block.c

```diff
@@ -115,6 +136,14 @@ struct vtblk_config {
 		uint32_t opt_io_size;
 	} vbc_topology;
 	uint8_t		vbc_writeback;
+	uint8_t		unused0[3];
```

Comment:
nit whitespace for struct members

Reply:
I think this is a GH artifact -- there are no spaces only tabs on that line.

Reply:
Oh.. you're looking at an older version :) All the whitespace nits should be fixed

Reply:
I miss gerrit :) 
Is there a clear way to tell which revision I am looking at?  Maybe clicking one of Mike's comments brought me to an older revision.  Sorry for the confusion!